mlflow/entities/assessment.py

```diff
@@ -96,6 +92,13 @@ def __post_init__(self):
                 "Expectations cannot have `error` specified.",
             )
 
+        # Set timestamp if not provided
+        current_time = int(time.time() * 1000)  # milliseconds
```

Comment:
nit: use helper defined below
```suggestion
        current_time = _get_current_time_ms()
```